Translations/it.lproj/Localizable.strings

```diff
@@ -1,10 +1,10 @@
 
 /*Name der App*/
-"app_name" = "Next Step";
+"app_name" = "Prossimo Passo";
```

Comment:
Not sure if we need to translate the original name of the application: French and German localization keep "Next Step" as the application name.